src/Libraries/RevitServices/Materials/Materials.cs

```diff
@@ -118,18 +116,14 @@ private void FindOrCreateDynamoErrorMaterial()
 
         private void FindDynamoGraphicsStyle()
         {
-            var styles = new FilteredElementCollector(DocumentManager.Instance.CurrentUIDocument.Document);
-            styles.OfClass(typeof(GraphicsStyle));
-
-            var gStyle = styles.ToElements().FirstOrDefault(x => x.Name == "Generic Models");
-
-            if (gStyle != null)
+            var genericModelCategory = Category.GetCategory(DocumentManager.Instance.CurrentDBDocument, BuiltInCategory.OST_GenericModel);
+            if(genericModelCategory != null)
             {
-                DynamoGStyleId = gStyle.Id;
-            }
-            else
-            {
-                DynamoGStyleId = ElementId.InvalidElementId;
+                var gStyle = genericModelCategory.GetGraphicsStyle(GraphicsStyleType.Projection);
```

Comment:
There are two style types for model category styles: projection & cut. Projection is mandatory. Cut is optional. It allows an element do be drawn with a different style when it is cut in some way compared to when it is only projected.